### PR TITLE
Add Qwen1.5-7B-Chat

### PR DESCRIPTION
**Pull request by ivy-lv11**
Add Qwen1.5-7B-Chat and the performance could be found in the issue https://github.com/analytics-zoo/nano/issues/1035.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -185,6 +185,7 @@ Over 20 models have been optimized/verified on `bigdl-llm`, including *LLaMA/LLa
 | RWKV5 |  | [link](python/llm/example/GPU/HF-Transformers-AutoModels/Model/rwkv5) |
 | Bark | [link](python/llm/example/CPU/PyTorch-Models/Model/bark) | [link](python/llm/example/GPU/PyTorch-Models/Model/bark) |
 | SpeechT5 |  | [link](python/llm/example/GPU/PyTorch-Models/Model/speech-t5) |
+| Qwen1.5-7B-Chat | [link](python/llm/example/CPU/HF-Transformers-AutoModels/Model/qwen1.5-7b) | [link](python/llm/example/GPU/HF-Transformers-AutoModels/Model/qwen1.5-7b) |
```
Comment (plusbang) [inline README.md:188]: Qwen1.5-7B-Chat -> Qwen1.5, and using `qwen1.5` as the example folder name may be better.

File: python/llm/example/CPU/HF-Transformers-AutoModels/Model/qwen1.5-7b/README.md [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
+<|im_start|>user
+AI是什么？<|im_end|>
+<|im_start|>assistant
+-------------------- Output --------------------
+<|im_start|>system
+You are a helpful assistant.<|im_end|>
+<|im_start|>user
+AI是什么？<|im_end|>
+<|im_start|>assistant人工智能（AI）是指计算机科学的一个分支，旨在开发能够执行通常需要人类智能的任务的算法和系统。这些任务包括但不限于理解自然语言、
+```
```
Comment (plusbang) [inline python/llm/example/CPU/HF-Transformers-AutoModels/Model/qwen1.5-7b/README.md:69]: Maybe we could also add sample output of English prompt.

File: README.md (hunk 1 above)
Comment (ivy-lv11) [reply README.md:188]: Yes, the folder name and the README.md have been updated.

File: python/llm/example/CPU/HF-Transformers-AutoModels/Model/qwen1.5-7b/README.md (hunk 2 above)
Comment (ivy-lv11) [reply python/llm/example/CPU/HF-Transformers-AutoModels/Model/qwen1.5-7b/README.md:69]: Sure, I will add sample output of English prompt.

File: python/llm/README.md [hunk 3]
```diff
@@ -81,7 +81,7 @@ Over 20 models have been optimized/verified on `bigdl-llm`, including *LLaMA/LLa
 | RWKV5 |  | [link](example/GPU/HF-Transformers-AutoModels/Model/rwkv5) |
 | Bark | [link](example/CPU/PyTorch-Models/Model/bark) | [link](example/GPU/PyTorch-Models/Model/bark) |
 | SpeechT5 |  | [link](example/GPU/PyTorch-Models/Model/speech-t5) |
-
+| Qwen1.5 | [link](python/llm/example/CPU/HF-Transformers-AutoModels/Model/qwen1.5) | [link](python/llm/example/GPU/HF-Transformers-AutoModels/Model/qwen1.5) |
```
Comment (plusbang) [inline python/llm/README.md:84]: The relative links are wrong.

Comment (ivy-lv11) [reply python/llm/README.md]: Sorry for the typo and I have fixed it.

**Review by plusbang — APPROVED** [review]
LGTM

File: python/llm/example/CPU/HF-Transformers-AutoModels/Model/qwen1.5/README.md [hunk 4]
```diff
@@ -0,0 +1,87 @@
...
+## Example: Predict Tokens using `generate()` API
+In the example [generate.py](./generate.py), we show a basic use case for a Qwen model to predict the next N tokens using `generate()` API, with BigDL-LLM INT4 optimizations.
+### 1. Install
+We suggest using conda to manage environment:
+```bash
+conda create -n llm python=3.9
+conda activate llm
+
+pip install --pre --upgrade bigdl-llm[all] # install bigdl-llm with 'all' option
+pip install transformers==4.37.2 # install the transformers which support Qwen2
```
Comment (Oscilloscope98) [inline python/llm/example/CPU/HF-Transformers-AutoModels/Model/qwen1.5/README.md:17]: Let's user transformers 4.37.0? qwen2_modelling.py did not changed from 4.37.0

Comment (ivy-lv11) [reply python/llm/example/CPU/HF-Transformers-AutoModels/Model/qwen1.5/README.md:17]: Sure, I will test the qwen1.5 on transformers 4.37.0

**Review by Oscilloscope98 — APPROVED** [review]
LGTM

**Pull request merged**